frontend/app/src/App.test.tsx

```diff
@@ -2181,19 +2189,21 @@ describe("App", () => {
 
       sendForwardMessage("newSession", { ...NEW_SESSION_JSON })
 
-      expect(clearInterval).toHaveBeenCalledWith(expect.any(Number))
-      expect(clearInterval).toHaveBeenCalledWith(expect.any(Number))
+      expect(clearInterval).toHaveBeenCalled()
+      expect(clearInterval).toHaveBeenCalled()
```

Comment:
**question:** Why not keep this syntax in vitest? `vitest` has `expect` available with the same semantics [[docs](https://vitest.dev/api/expect.html#expect-any)]

Reply:
It's tied to the timeouts...in a browser. setInterval is a number, in node js (where vitest runs) the timeout is a Timeout object. I tried modifying it to be an instance of the class, but it failed.

I can modify it to `expect(clearInterval).toHaveBeenCalledWith(expect.any(Object))`

I started to just think it wasn't as important with these differences, so just took out the With call. Let me know if you have any thoughts or a preference. I'm indifferent here.